pod/locale/fr/LC_MESSAGES/django.po

```diff
@@ -9816,6 +9830,14 @@ msgstr "Chapitrer la vidéo"
 msgid "Dress the video"
 msgstr "Habiller la vidéo"
 
+#: pod/video/templates/videos/link_video_dropdown_menu.html
+msgid "Duplicate the form"
+msgstr "Dupliquer la vidéo"
+
+#: pod/video/templates/videos/link_video_dropdown_menu.html
+msgid "Are you sure you want to duplicate this form?"
+msgstr "Voulez-vous vraiment dupliquer la vidéo ?"
```

Comment:
pour être au plus pres du texte d'origine, il faudrait mettre "cette vidéo" au lieu de "la vidéo", et corriger "form" dans le texte d'origine (ou mettre "the video" en texte d'origine)